content/platform/_index.md

```diff
@@ -24,23 +25,41 @@ sections:
   - block: markdown
     id: numbers
     content:
-      title: The research lifecycle we enable.
-      subtitle: Interactive computing hubs provide tools that facilitate the entire knowledge creation and sharing lifecycle.
+      title: Hubs enable the research lifecycle with open tools and services
+      subtitle: Interactive computing hubs integrate open tools and services to provide a digital home that takes users from learning their first workflows to making discoveries and sharing with others.
       text: |
 
         {{< figure src="/images/service/research-lifecycle.png" alt="The research lifecycle we enable.">}}
 
+        <center> <strong>Open source tools we use and support in our service. See below for some of the workflows these tools enable.</strong></center>
+
+        <div class="container logos">
+            <div class="row justify-content-center">
+                <div class="section-heading d-flex flex-wrap justify-content-center col-12 mb-3 text-center">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/jupyterlab.svg" alt="Jupyter Lab logo">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/jupyterhub.svg" alt="Jupyter Hub logo">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/jupyterbook.svg" alt="Jupyter Book logo">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/jupyter.svg" alt="Jupyter logo">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/myst.svg" alt="MyST logo">
+                    <img class="logo col-6 col-md-4" src="/images/logos/project/binder.svg" alt="Binder logo">
+                </div>
+            </div>
+        </div>
+
   - block: markdown
     id: magiclink
     design:
         columns: 2
     content:
-      title: Magic links let you instantly share content on a hub.
+      title: Magic links let you instantly share content on a hub
       subtitle:  Clicking a magic link to instantly give a user their own copy of your content so that they can interact and explore in a live environment.
       text: |
 
         {{< video src="videos/magic-links.mp4">}}
-
+        
+        <p class="figcaption">
+          Powered by <a href="https://mystmd.org"><img src="/images/logos/project/myst.svg" /></a> and <a href="https://jupyterhub.readthedocs.io"><img src="/images/logos/project/jupyterhub.svg" /></a>
+        </p>
```

Comment:
This figcaption is rendering pretty huge for me, I think you might need to define the width and height of the SVGs.

<img width="1057" alt="Screenshot 2024-07-03 at 11 07 52" src="https://github.com/2i2c-org/2i2c-org.github.io/assets/45105935/a8541cac-3a39-4d17-9756-c9d93ebfcef0">